### DISCUSSION TITLE
Does not exist video recording Practical Statistics (PS) - Zeus Night Class

### DISCUSSION THREAD

**Discussion by FarhadFamser99** [teamalgoritma/community#921] [Q&A] [answered]
Selamat siang kak, 
untuk materi Practical Statistics (PS) pada zeus night class 
tanggal 23 - 24 september 2024 
video recording nya does not exist.
padahal tertulis di google classrom nya Exp. Date: 2024-10-01
boleh minta bantuannya kak udah di resolve.

**Comment by adizaen** [comment, marked as answer]
Dear @FarhadFamser99 

Selamat pagi Pak Farhad. Untuk rekaman sudah bisa diakses kembali ya Pak sampai batas waktu yang telah ditentukan di google classroom. Kami mohon maaf karena ada kendala teknis dari kami sehingga recording tidak bisa diakses. Terima kasih banyak Pak atas perhatiannya.

Best regards,
Team Algoritma
